README.md

```diff
@@ -193,6 +193,7 @@ Approvers ([@open-telemetry/js-approvers](https://github.com/orgs/open-telemetry
 Maintainers ([@open-telemetry/js-maintainers](https://github.com/orgs/open-telemetry/teams/javascript-maintainers)):
 
 - [Daniel Dyla](https://github.com/dyladan), Dynatrace
+- [Chengzhong Wu](https://github.com/legendecas), Alibaba
```

Comment:
Alphabetical order?

Reply:
@dyladan not sure if it was intentional but i think we should go with alpha order too